deployment/data-streams/changeset/testutil/test_helpers.go

```diff
@@ -57,3 +61,72 @@ func NewMemoryEnv(t *testing.T, deployMCMS bool, optionalNumNodes ...int) deploy
 
 	return env
 }
+
+func DeployMCMS(
+	t *testing.T,
+	e deployment.Environment,
+	addrs ...map[uint64][]common.Address,
+) (env deployment.Environment, mcmsState *commonChangesets.MCMSWithTimelockState, timelocks map[uint64]*proposalutils.TimelockExecutionContracts) {
+	t.Helper()
+
+	chainSelector := TestChain.Selector
+	config := proposalutils.SingleGroupMCMSV2(t)
+
+	env, err := commonChangesets.Apply(t, e, nil,
+		commonChangesets.Configure(
+			deployment.CreateLegacyChangeSet(commonChangesets.DeployLinkToken),
```

Comment:
Do we need to deploy link token here? Or this is just for reward manager? I'd keep it separate - checkout test_helpers.go for fee manager optionally accepts input params to select what to deploy. I think if we move all changesets into 1 package we'll benefit from a common env setup pattern than we're doing now

Reply:
Actually we don't need it.
I do agree about changesets, created ticket [here](https://smartcontract-it.atlassian.net/browse/DPA-1640).